lib/src/main/scala/spinal/lib/Stream.scala

```diff
@@ -68,7 +70,7 @@ class EventFactory extends MSFactory {
   }
 }
 
-class Stream[T <: Data](val payloadType :  HardType[T]) extends Bundle with IMasterSlave with DataCarrier[T] {
+class Stream[T <: Data](val payloadType :  HardType[T]) extends Bundle with FormalMasterSlave with DataCarrier[T] {
```

Comment:
Is that possible to add new traits but change the existing running way? I think traits itself is design for decoupling things. If I am correct, the definition of FormalMasterSlave traits does not use the methods bring from IMasterSlave.